lib/ret/session_token.ex

```diff
@@ -0,0 +1,35 @@
+defmodule Ret.SessionToken do
```

Comment:
We have a few of these JWT tokens now, I think we should add a one or two line comment at the top of all these describing them so we can keep them straight